data/data/baremetal/masters/main.tf

```diff
@@ -41,7 +41,9 @@ resource "ironic_deployment" "openshift-master-deployment" {
     count.index,
   )
 
-  instance_info = var.instance_infos[count.index]
-  user_data     = var.ignition
+  instance_info 	= var.instance_infos[count.index]
+  user_data     	= var.ignition
```

Comment:
I think we need to remove the existing user_data line here?

Reply:
But leaving it would mean that we can still send the pointer ignition by default, yes? If we have the `user_data_url` in addition to the `user_data`, then we pick the former.

Or. Do you prefer we completely do away with `user_data`?